cmd/clusterctl/client/config/providers_client.go

```diff
@@ -70,6 +70,7 @@ const (
 	KubeadmBootstrapProviderName    = "kubeadm"
 	TalosBootstrapProviderName      = "talos"
 	MicroK8sBootstrapProviderName   = "microk8s"
+	OCNEBootstrapProviderName       = "ocne"
```

Comment:
nit:
What about
```suggestion
	OracleNativeBootstrapProviderName       = "ocne"
```
as a variable name, this makes reading the code easier given that this abbreviation is not so common...
(same for the control plane provider)

Reply:
@fabriziopandini i updated the name to `OracleCloudNativeBootstrapProviderName` . Same for control plane.